config/default/common/config/wv.json/categories/science_disciplines/Terrestrial Hydrosphere.json

```diff
@@ -1,6 +1,6 @@
 {
     "categories": {
-        "scientific": {
+        "science disciplines": {
```

Comment:
We probably should set up some JSON style linter (I haven't don't this before but my vscode does some correcting 🤷 ). I [don't think you want spaces for property names](https://stackoverflow.com/a/19287394/4589331) although it does work and we probably have them throughout the codebase such as is in the line below.

I think it would be more stylistically correct to say something like
```
categories": [
  {
    id: "cat_1",
    name: "science disciplines",
    someOtherName: 'some value'
  }
]
```

Reply:
Hmmm yeah this is a good point.  Didn't really cross my mind since we are already violating this rule elsewhere (e.g. "hazards and disasters").  I wonder what the real consequences of having a space here might actually be for us?  I could see needing to avoid spaces if we were generating some JSON as part of an API response that would be consumed by different machines/operating systems/environments that may have different expectations of formatting.  

JSON linting would be nice, especially based on a JSON schema.  I think having a JSON schema for these kinds of config files could go a long way towards helping third parties better understand what the expected content, and meaning of these configs.  Perhaps it could also help streamline adding new layers for the uninitiated.

I will add a topic to the next tech dev meeting agenda!